.ruff.toml.jinja

```diff
@@ -1,4 +1,10 @@
-target-version = "py310"
+target-version = {% if odoo_version < 14 -%}
```

Comment:
Shouldn't all this be in the same line?

Reply:
I just thought it'd be more readable like this.

I added `-` to the jinja markers, to make sure it strips whitespace.

Reply:
It still seems incorrect.

Reply:
I changed it to one line.